<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>519</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Virtual Box++ Settings</string>
  </property>
  <layout class="QGridLayout" name="centralGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="centralTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="filesaAndDirectories">
      <attribute name="icon">
       <iconset resource="VirtualBoxPlusPlus.qrc">
        <normaloff>:/Resources/Images/FOLDER.png</normaloff>:/Resources/Images/FOLDER.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Files and directories</string>
      </attribute>
      <layout class="QGridLayout" name="homeGridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="commandLabel">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Commands:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="pluginLineEdit">
         <property name="minimumSize">
          <size>
           <width>570</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="vboxmLabel">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>VBoxManage.exe:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="additionalsLabel">
         <property name="text">
          <string>Additional directories:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="pluginLabel">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Plugins:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="commandLineEdit">
         <property name="minimumSize">
          <size>
           <width>570</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="homeLineEdit">
         <property name="minimumSize">
          <size>
           <width>570</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="homePushButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="playlistLabel">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Playlists:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pluginPushButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="vboxmanageLineEdit">
         <property name="minimumSize">
          <size>
           <width>570</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="homeDirLabel">
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Home:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="vboxmanagePushButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="playlistLlineEdit">
         <property name="minimumSize">
          <size>
           <width>570</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QTabWidget" name="additionalDirectoriesTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pluginDirectories">
          <attribute name="icon">
           <iconset resource="VirtualBoxPlusPlus.qrc">
            <normaloff>:/Resources/Images/PLUGIN.png</normaloff>:/Resources/Images/PLUGIN.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Plugin directories</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="pluginDirsTableWidget">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="playlistDirectories">
          <attribute name="icon">
           <iconset resource="VirtualBoxPlusPlus.qrc">
            <normaloff>:/Resources/Images/PLAYLIST.png</normaloff>:/Resources/Images/PLAYLIST.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Playlist directories</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableWidget" name="playlistDirsTableWidget">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="commandPushButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="playlistPushButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="metricsAndMisc">
      <attribute name="icon">
       <iconset resource="VirtualBoxPlusPlus.qrc">
        <normaloff>:/Resources/Images/SETTINGS.png</normaloff>:/Resources/Images/SETTINGS.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Other configurations</string>
      </attribute>
      <widget class="QGroupBox" name="metricsGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>390</width>
         <height>160</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>390</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>390</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>Metrics</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="sampleIntervalLlabel">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Sample interval (seconds):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sampleIntervalSpinBox">
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="maxNumberSamplesLabel">
          <property name="text">
           <string>Max number of samples:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sampleCountSpinBox">
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="selectionTimePeriodLabel">
          <property name="text">
           <string>Your selection covers a period of {0} seconds or {1} minutes</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="metricsFromStartupRadioButton">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="metricsFromStartupLabel">
          <property name="text">
           <string>Gather metrics from startup:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>558</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="controlButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>vboxmanageLineEdit</tabstop>
  <tabstop>vboxmanagePushButton</tabstop>
  <tabstop>homeLineEdit</tabstop>
  <tabstop>homePushButton</tabstop>
  <tabstop>playlistLlineEdit</tabstop>
  <tabstop>playlistPushButton</tabstop>
  <tabstop>pluginLineEdit</tabstop>
  <tabstop>pluginPushButton</tabstop>
  <tabstop>additionalDirectoriesTabWidget</tabstop>
  <tabstop>playlistDirsTableWidget</tabstop>
  <tabstop>pluginDirsTableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="VirtualBoxPlusPlus.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>controlButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>controlButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
